### ISSUE TITLE
Black BackgroundColor Crashes App

### ISSUE THREAD

**Issue by marckassay** [timbru31/cordova-plugin-lottie-splashscreen#67]
I noticed, either one of the following works without crashing app:
```xml
<preference name="LottieBackgroundColor" value="#100000a3" />
<preference name="LottieBackgroundColor" value="#101010a3" />
```

But neither one of these work. When launching it crashes app.
```xml
<preference name="LottieBackgroundColor" value="#000000a3" />
<preference name="LottieBackgroundColor" value="#010101a3" />
```

**Comment by timbru31** [comment]
Thanks, I'll take a look. Which platform(s) do crash?

**Comment by marckassay** [comment]
My device has Android Pie. Here is more, info:
```shell
 ionic info

Ionic:

   ionic (Ionic CLI)             : 4.12.0 (C:\Users\marck\AppData\Roaming\nvm\v11.0.0\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.1.2
   @angular-devkit/build-angular : 0.13.7
   @angular-devkit/schematics    : 7.2.4
   @angular/cli                  : 7.3.7
   @ionic/angular-toolkit        : 1.4.1

Cordova:

   cordova (Cordova CLI) : 9.0.0
   Cordova Platforms     : android 8.0.0
   Cordova Plugins       : cordova-plugin-ionic-webview 3.1.2, (and 9 other plugins)

System:

   Android SDK Tools : 26.1.1 (C:\Users\marck\AppData\Local\Android\Sdk)
   NodeJS            : v11.0.0 (C:\Program Files\nodejs\node.exe)
   npm               : 6.9.0
   OS                : Windows 10
```

**Comment by timbru31** [comment]
Thanks, that helps to debug 👍

**Comment by timbru31** [comment]
Indeed the following error is thrown in Logcat:

```
     Caused by: java.lang.IllegalArgumentException: Unknown color
        at de.dustplanet.cordova.lottie.ColorHelperKt.parseColor(ColorHelper.kt:32)
        at de.dustplanet.cordova.lottie.LottieSplashScreen$createView$1.run(LottieSplashScreen.kt:102)
        at android.app.Activity.runOnUiThread(Activity.java:6282)
        at de.dustplanet.cordova.lottie.LottieSplashScreen.createView(LottieSplashScreen.kt:69)
        at de.dustplanet.cordova.lottie.LottieSplashScreen.createView$default(LottieSplashScreen.kt:68)
        at de.dustplanet.cordova.lottie.LottieSplashScreen.pluginInitialize(LottieSplashScreen.kt:20)
        at org.apache.cordova.CordovaPlugin.privateInitialize(CordovaPlugin.java:58)
        at org.apache.cordova.PluginManager.getPlugin(PluginManager.java:171)
        at org.apache.cordova.PluginManager.startupPlugins(PluginManager.java:97)
        at org.apache.cordova.PluginManager.init(PluginManager.java:86)
        at org.apache.cordova.CordovaWebViewImpl.init(CordovaWebViewImpl.java:117)
        at org.apache.cordova.CordovaActivity.init(CordovaActivity.java:149)
        at org.apache.cordova.CordovaActivity.loadUrl(CordovaActivity.java:224)
        at com.example.hello.MainActivity.onCreate(MainActivity.java:39)
        at android.app.Activity.performCreate(Activity.java:7136)
        at android.app.Activity.performCreate(Activity.java:7127)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
```

**Comment by timbru31** [comment]
You are experiencing the following Android (more precise AAPT2) bug: https://issuetracker.google.com/issues/116700688
It's broken in Android Studio 3.3, but already fixed in Android Studio 3.4 (in RC phase) and 3.5.  
Either install those preview versions or wait until Android Studio 3.4 reaches the stable channel.  

Then you can add the following to your `build-extras.gradle` file:
```gradle
android {
    aaptOptions {
        additionalParameters "--keep-raw-values"
    }
}
```

Please not that you need at least the version 3.4.0-rc03 of the Android Gradle Plugin (AGP), too. You can change it e.g. with this answer: https://stackoverflow.com/questions/17727645/how-to-update-gradle-in-android-studio  
Sadly you need to upgrade the Gradle version to >= 5, too.

Let me know if you need more assistance. Closing this issue, since it's not my bug.

**Comment by marckassay** [comment]
Nice work! It's not urgent for me to upgrade just yet, but when I do I'll reference back to this post. Thanks!
